plugins/inventory/iocage.py

```diff
@@ -71,6 +71,15 @@
               - Enable O(sudo_preserve_env) if O(sudo) is enabled.
             type: dict
             default: {}
+        hooks:
+            description:
+              - List of paths to the hook files in a jail.
+              - Content of the hook files is stored in the items of the list C(iocage_hooks).
+              - If a hook file is not available the item keeps the dash character C(-).
+              - The variable C(iocage_hooks) is not created if O(hooks) is empty.
+            type: list
+            elements: path
+            version_added: 10.3.0
```

Comment:
10.3.0 is already out.
```suggestion
            version_added: 10.4.0
```